src/appengine/handlers/cron/fuzzer_and_job_weights.py

```diff
@@ -20,19 +20,32 @@
 from base import utils
 from datastore import data_handler
 from datastore import data_types
+from datastore import ndb
 from datastore import ndb_utils
 from google_cloud_utils import big_query
 from handlers import base_handler
 from libs import handler
 from metrics import fuzzer_stats
 from metrics import logs
+from system import environment
 
 QuerySpecification = collections.namedtuple(
     'QuerySpecification', ['query_format', 'formatter', 'reason'])
 
 SpecificationMatch = collections.namedtuple('SpecificationMatch',
                                             ['new_weight', 'reason'])
 
+DEFAULT_MULTIPLIER = 10.0  # Used for blackbox and unrun jobs.
+DEFAULT_SANITIZER_WEIGHT = 0.1
+
+SANITIZER_WEIGHTS = {
```

Comment:
these are up for discussion/debate.

Reply:
looks reasonable, though I wouldn't mind equalizing MSan and UBSan

Reply:
> We currently don't give architectures special consideration. I can add a TODO here.
> 

Another thought that is probably a topic for future discussion, but I suspect the "optimal" amount that libFuzzer/x64/ASAN should be run is lower when there are jobs like AFL/x64/ASAN and libFuzzer/x64/ASAN since they will find many of the same bugs. Same is true for other libFuzzer jobs (we get similar mutations, corpus elements from LF/UBSAN as we do from LF/ASAN so perhaps AFL/ASAN should be run more frequently than LF/ASAN).

> What do you mean re non-blackbox vs blackbox jobs? Currently we assign a default of 10 to those, but I expect that we need to set most of them manually.

Sorry the question didn't make much sense. I'm asking if this CL will affect the relative weighting of coverage-guided to blackbox fuzzing, ie: will libFuzzer/AFL get run a lot more/less compared than it is now.